dlgs.go

```diff
@@ -46,21 +46,26 @@ func (b *MsgBuilder) YesNo() bool {
 	return b.yesNo()
 }
 
+/* OkDialog spawns the message dialog with an information icon and single button, "Ok". */
+func (b *MsgBuilder) OkDialog() {
```

Comment:
This function should just be called `Ok` for consistency with the other functions, and to avoid redundancy in the "one-liner" use case:

    dialog.Message("Hello world!").Ok()

Reply:
ok, I'll rename it

Reply:
Maybe Info() would be a better name come to think of it?